### PR TITLE
deps:install statsmodel

### PR DESCRIPTION
**Pull request by Dineshjnld**
for: #4

### CODE REVIEW

File: docs/dinesh/1_install_ml_deps.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+## Problem:
+Encountered dependency missing error 
+```shell
+module not found- statsmodel
+```
+## Solution:
+
+1. `poetry add --group ml statsmodels`
+2. `poetry run streamlit run app.py`
+
```
Comment (Nasfame) [inline docs/dinesh/1_install_ml_deps.md:12]: Documented the solution

Comment (Nasfame) [reply docs/dinesh/1_install_ml_deps.md:12]: Good job

**Review by Nasfame — CHANGES_REQUESTED** [review]
Create a folder inputs and move the input archives

File: docs/dinesh/1_install_ml_deps.md (hunk 1 above)
Comment (Dineshjnld) [reply docs/dinesh/1_install_ml_deps.md:12]: done

**Review by Nasfame — APPROVED** [review]
LGTM

**Pull request merged**